README.md

```diff
@@ -8,3 +8,17 @@ It covers:
   -   Add files to be tracked by Git `git add 'my-file'`.
   -   Stage changes to be commited `git add 'my-file'`.
   -   Commit changes with appropriate commit message `git commit -m 'commit message'`.
+  - Finding out information regarding what files are modified and those that are in the staging area using `git status`.
```

Comment:
My opinion is of this line being between tracking and committing a file. Kindly look at it and provide your thoughts .

Reply:
That sounds like a suitable positioning for it. Change to be implemented in the next PR